### DISCUSSION TITLE
关于 MSYS2 和 Cygwin 环境的检测

### DISCUSSION THREAD

**Discussion by Yangmoooo** [RubyMetric/chsrc#136] [Q&A]
在 #133 中有关 MSYS2 和 Cygwin 环境的 [讨论](https://github.com/RubyMetric/chsrc/issues/133#issuecomment-2525132261)，其实我很好奇为什么要对它们进行单独处理。既然两者都支持运行 Windows 程序，为什么不统一使用 Windows 的 `curl.exe` 呢？

**Comment by ccmywish** [comment]
你提出来的是一个好问题，我新开了一个 issue #140
